src/components/cylc/TaskFilterSelect.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+<v-select
+  :label="`Filter by ${type}`"
+  :data-cy="`filter ${type}`"
+  :items="items"
+  clearable
+  multiple
+  :placeholder="`Filter by ${type}`"
+  v-model="localValue.states"
+>
+<template v-slot:item="{ item, props }">
+  <v-list-item v-bind="props" :title="undefined">
+    <Workflowicon v-if="type==='workflow state'" :status=item.raw />
+    <Task v-if="type==='task state'" :task="{ state: item.raw }" />
+    <span class="ml-2">{{ item.raw }}</span>
+  </v-list-item>
+</template>
+<template v-slot:selection="{ item, index }">
+  <v-chip closable @click:close="removeItem(item.raw)" v-if="index >= 0 && index < maxVisibleStates">
+    <template v-slot:prepend>
+      <Workflowicon v-if="type==='workflow state'" :status=item.raw />
+      <Task v-if="type==='task state'" :task="{ state: item.raw }" />
+    </template>
+    {{item.title}}
+
+  </v-chip>
+  <span
+      v-if="index === maxVisibleStates"
+      class="text-grey text-caption"
+    >
+      (+{{ localValue.states.length - maxVisibleStates }})
+  </span>
+</template>
+</v-select>
```

Comment:
There is some missing indentation here

Reply:
Yet linting doesn't complain?

Reply:
#1536 - it's included in `eslint-plugin-vue` but only if you turn on "strongly-recommended"